src/devices/sound/ymfm.cpp

```diff
@@ -125,6 +352,17 @@
 //
 
 
+//*********************************************************
+//  MACROS
+//*********************************************************
+
+// macro to encode four operator numbers into a 32-bit value in the
+// operator maps for each register class
+#define YMFM_OP4(o1,o2,o3,o4) u32(o1 | (o2 << 8) | (o3 << 16) | (o4 << 24))
+#define YMFM_OP2(o1,o2) YMFM_OP4(o1,o2,0xff,0xff)
+#define YMFM_OP0() YMFM_OP4(0xff,0xff,0xff,0xff)
```

Comment:
Any reason not to use `constexpr` functions for these?  It will avoid the potential precedence issues in `YMFM_OP4` depending on what kind of expression is passed for the arguments.